bundles/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComLighting1Message.java

```diff
@@ -85,6 +87,8 @@ public byte toByte() {
     public Commands command;
     public boolean group;
 
+    private static byte[] lastUnit = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
I doubt that this state statefulness will fly with multiple bridges or multiple unrelated devices, wdyt?